pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -659,14 +659,15 @@ type AzureCloudSpec struct {
 	ClientID       string `json:"clientID,omitempty"`
 	ClientSecret   string `json:"clientSecret,omitempty"`
 
-	ResourceGroup         string `json:"resourceGroup"`
-	VNetResourceGroup     string `json:"vnetResourceGroup"`
-	VNetName              string `json:"vnet"`
-	SubnetName            string `json:"subnet"`
-	RouteTableName        string `json:"routeTable"`
-	SecurityGroup         string `json:"securityGroup"`
-	AssignAvailabilitySet *bool  `json:"assignAvailabilitySet"`
-	AvailabilitySet       string `json:"availabilitySet"`
+	ResourceGroup           string `json:"resourceGroup"`
+	VNetResourceGroup       string `json:"vnetResourceGroup"`
+	VNetName                string `json:"vnet"`
+	SubnetName              string `json:"subnet"`
+	RouteTableName          string `json:"routeTable"`
+	SecurityGroup           string `json:"securityGroup"`
+	NodePortsAllowedIPRange string `json:"nodePortsAllowedIPRange,omitempty"`
```

Comment:
Migration (copying from old types to new) is missing. You can follow one of two options:
* drop new API types changes (including generated CRDs and chart)
* add missing field (here, and for other changed types from this PR) to the migration (see https://github.com/kubermatic/kubermatic/blob/master/pkg/install/crdmigration/clones.go#L452)